helm/sematic-server/templates/cleaner-cron.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-cleaner
+  labels:
+    app.kubernetes.io/component: cleaner
+spec:
+  schedule: {{ .Values.cleaner.crontab }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: {{ .Values.cleaner.ttlSecondsAfterFinished }}
+      template:
+        spec:
+          {{- with .Values.image.pull_secrets }}
+          imagePullSecrets:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          serviceAccountName: {{ .Values.service_account.name | default .Release.Name }}
+          containers:
+          - name: cleaner
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+            imagePullPolicy: {{ .Values.image.pull_policy }}
+            env:
+            - name: SEMATIC_API_ADDRESS
+              value: "http://{{ .Release.Name }}"
+            - name: RUNNING_AS_CRON_JOB
```

Comment:
may be better to make this more explicit as `RUNNING_AS_CLEANUP_CRON_JOB` in case we need to add other types of cronjobs in the future.

Reply:
done